### DISCUSSION TITLE
Lodestar Standup 2024-8 (February 27, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6467] [Standups]
### Meeting Info

[Feb 27, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/feb-27-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.16.1 hotfixes. Inclusions necessary for Deneb success?
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
I would like to see the feasibility of us being able to contribute with Nimbus on more light client initiatives. The [latest roadmap proposal by Etan](https://hackmd.io/@etan-status/electra-lc) here may hopefully generate some ideas and allow us to contribute by including some of these items as part of a proof of concept to push some advancement in light client features.

Floated an idea with Gajinder to have Lodestar breakout calls on light client implementation if justified and can have someone lead this effort.

**Reply by jeluard** [reply]
Definitely looking forward to push that initiative. This roadmap is quite low-level and tech oriented, it might help to discuss higher level goals too.

**Reply by philknows** [reply]
Definitely. After some discussion with @g11tech , I'm also thinking this might be worth ice-boxing until some clearer vision for this part of the protocol is discussed. Can still bring it up on standup though. Please feel free to include your thoughts if you have any, but I want to make sure the changes proposed actually lead to an end goal of builders actually using the protocol for something tangible from an Ethereum UX/use-case perspective. There's so many things to focus on in protocol, but where we get the most value for time is important.

**Comment by nazarhussain** [comment]
We should cleanup old stale PRs on all repos. 

<img width="551" alt="image" src="https://github.com/ChainSafe/lodestar/assets/112468/716f2030-db10-429b-be6d-3defe8d327d1">

**Reply by philknows** [reply]
Definitely agree on this. Can lead this initiative async. But we can discuss on the call what other repositories should be included here.

**Comment by nflaig** [comment]
We might also quickly discuss how we wanna handle removing support for older node versions. I think it is good to still support v18.x to not force existing users to upgrade immediately but we need to make sure Lodestar actually works on that version. For that, I'd suggest we run at least unit tests against two node versions (18, 20) as those are quite fast anyways.

**Comment by philknows** [comment]
Notes located here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#february-27-2024-planning-and-standup-meeting

Closing in favour of #6493
